### ISSUE TITLE
How Do I Debug xUnit.NET Tests in Monodevelop?

### ISSUE THREAD

**Issue by vintzl** [mono/monodevelop#8286]
Monodevelop Version: 7.8.4.1-0xamarin6+debian10b1
How can I debug Unit Tests with MonoDevelop?

**Comment by usr-sse2** [comment]
@vintzl, please describe your problem with more details. Before migrating to NUnit I used debugging xUnit tests in MonoDevelop or VS for Mac, and it worked (although not always).

**Comment by vintzl** [comment]
Well, I cannot see nowhere, in the UI, how to clearly debug them.

If I do "right click" in one of the Unit Tests, I have no contextual menu allowing debugging…

Finally, if I try to debug all of them, from the "play button" , no break points are respected, and no test result is return, as if MonoDevelop just builds them.

**Comment by usr-sse2** [comment]
It seems that you are trying to debug a .NET Core project on Linux. Microsoft .NET Core debugger `vsdbg` is included only in Visual Studio for Mac because of its license. This issue is not specific to unit tests. You should either convert your solution to .NET Framework and use Mono debugger, or use Samsung .NET Core debugger extension (which doesn't work for me – it can't even start `dotnet`, because it tries to execute `dotnet-/usr/local/share/dotnet/dotnet` command).

**Comment by vintzl** [comment]
Thanks.

> It seems that you are trying to debug a .NET Core project on Linux. Microsoft .NET Core debugger `vsdbg` is included only in Visual Studio for Mac because of its license. This issue is not specific to unit tests. You should either convert your solution to .NET Framework and use Mono debugger, or use Samsung .NET Core debugger extension (which doesn't work for me – it can't even start `dotnet`, because it tries to execute `dotnet-/usr/local/share/dotnet/dotnet` command).

And what about:
`sudo ln -s /usr/share/dotnet/dotnet /usr/local/share/dotnet/dotnet`

I have found this issue:
- https://github.com/dotnet/core/issues/505

Now I am in trouble :s My project is too big for VSCode and its OmniSharp (lags, slow, crashes…) :s

**Comment by usr-sse2** [comment]
No, the problem is not `/local/` (I was writing from memory). The problem is that it runs
```
dotnet-/usr/share/dotnet/dotnet
```
instead of
```
/usr/share/dotnet/dotnet
```

**Comment by vintzl** [comment]
@ usr-sse2 I have successfully made the Samsung debugger working (.NET Core ≥ 2.1).
Question:
- Why not making this debugger part of MonoDevelop?
